Modbus node stopped working after 11.0.1

Modbus stopped working after the latest update of Node Red.
got this from the logg :

14 Feb 14:51:04 - [warn] [node-red-contrib-modbus/Modbus-Flex-Getter] Error: Cannot find module 'source-map-support'
Require stack:
- /opt/node_modules/node-red-contrib-modbus/modbus/modbus-flex-getter.js
- /opt/node_modules/@node-red/registry/lib/loader.js
- /opt/node_modules/@node-red/registry/lib/index.js
- /opt/node_modules/@node-red/runtime/lib/nodes/index.js
- /opt/node_modules/@node-red/runtime/lib/index.js
- /opt/node_modules/node-red/lib/red.js
- /opt/node_modules/node-red/red.js
14 Feb 14:51:04 - [warn] [node-red-contrib-modbus/Modbus-Write] Error: Cannot find module 'source-map-support'
Require stack:
- /opt/node_modules/node-red-contrib-modbus/modbus/modbus-write.js
- /opt/node_modules/@node-red/registry/lib/loader.js
- /opt/node_modules/@node-red/registry/lib/index.js
- /opt/node_modules/@node-red/runtime/lib/nodes/index.js
- /opt/node_modules/@node-red/runtime/lib/index.js
- /opt/node_modules/node-red/lib/red.js
- /opt/node_modules/node-red/red.js
14 Feb 14:51:04 - [warn] [node-red-contrib-modbus/Modbus-Flex-Write] Error: Cannot find module 'source-map-support'
Require stack:
- /opt/node_modules/node-red-contrib-modbus/modbus/modbus-flex-write.js
- /opt/node_modules/@node-red/registry/lib/loader.js
- /opt/node_modules/@node-red/registry/lib/index.js
- /opt/node_modules/@node-red/runtime/lib/nodes/index.js
- /opt/node_modules/@node-red/runtime/lib/index.js
- /opt/node_modules/node-red/lib/red.js
- /opt/node_modules/node-red/red.js
14 Feb 14:51:04 - [warn] [node-red-contrib-modbus/Modbus-Server] Error: Cannot find module 'source-map-support'
Require stack:
- /opt/node_modules/node-red-contrib-modbus/modbus/modbus-server.js
- /opt/node_modules/@node-red/registry/lib/loader.js
- /opt/node_modules/@node-red/registry/lib/index.js
- /opt/node_modules/@node-red/runtime/lib/nodes/index.js
- /opt/node_modules/@node-red/runtime/lib/index.js
- /opt/node_modules/node-red/lib/red.js
- /opt/node_modules/node-red/red.js
14 Feb 14:51:04 - [warn] [node-red-contrib-modbus/Modbus-Flex-Server] Error: Cannot find module 'source-map-support'
Require stack:
- /opt/node_modules/node-red-contrib-modbus/modbus/modbus-flex-server.js
- /opt/node_modules/@node-red/registry/lib/loader.js
- /opt/node_modules/@node-red/registry/lib/index.js
- /opt/node_modules/@node-red/runtime/lib/nodes/index.js
- /opt/node_modules/@node-red/runtime/lib/index.js
- /opt/node_modules/node-red/lib/red.js
- /opt/node_modules/node-red/red.js
14 Feb 14:51:04 - [warn] [node-red-contrib-modbus/Modbus-Queue-Info] Error: Cannot find module 'source-map-support'
Require stack:
- /opt/node_modules/node-red-contrib-modbus/modbus/modbus-queue-info.js
- /opt/node_modules/@node-red/registry/lib/loader.js
- /opt/node_modules/@node-red/registry/lib/index.js
- /opt/node_modules/@node-red/runtime/lib/nodes/index.js
- /opt/node_modules/@node-red/runtime/lib/index.js
- /opt/node_modules/node-red/lib/red.js
- /opt/node_modules/node-red/red.js
14 Feb 14:51:04 - [warn] [node-red-contrib-modbus/Modbus-Flex-Connector] Error: Cannot find module 'source-map-support'
Require stack:
- /opt/node_modules/node-red-contrib-modbus/modbus/modbus-flex-connector.js
- /opt/node_modules/@node-red/registry/lib/loader.js
- /opt/node_modules/@node-red/registry/lib/index.js
- /opt/node_modules/@node-red/runtime/lib/nodes/index.js
- /opt/node_modules/@node-red/runtime/lib/index.js
- /opt/node_modules/node-red/lib/red.js
- /opt/node_modules/node-red/red.js
14 Feb 14:51:04 - [warn] [node-red-contrib-modbus/Modbus-IO-Config] Error: Cannot find module 'source-map-support'
Require stack:
- /opt/node_modules/node-red-contrib-modbus/modbus/modbus-io-config.js
- /opt/node_modules/@node-red/registry/lib/loader.js
- /opt/node_modules/@node-red/registry/lib/index.js
- /opt/node_modules/@node-red/runtime/lib/nodes/index.js
- /opt/node_modules/@node-red/runtime/lib/index.js
- /opt/node_modules/node-red/lib/red.js
- /opt/node_modules/node-red/red.js
14 Feb 14:51:04 - [warn] [node-red-contrib-modbus/Modbus-Response-Filter] Error: Cannot find module 'source-map-support'
Require stack:
- /opt/node_modules/node-red-contrib-modbus/modbus/modbus-response-filter.js
- /opt/node_modules/@node-red/registry/lib/loader.js
- /opt/node_modules/@node-red/registry/lib/index.js
- /opt/node_modules/@node-red/runtime/lib/nodes/index.js
- /opt/node_modules/@node-red/runtime/lib/index.js
- /opt/node_modules/node-red/lib/red.js
- /opt/node_modules/node-red/red.js
14 Feb 14:51:04 - [warn] [node-red-contrib-modbus/Modbus-Flex-Sequencer] Error: Cannot find module 'source-map-support'
Require stack:
- /opt/node_modules/node-red-contrib-modbus/modbus/modbus-flex-sequencer.js
- /opt/node_modules/@node-red/registry/lib/loader.js
- /opt/node_modules/@node-red/registry/lib/index.js
- /opt/node_modules/@node-red/runtime/lib/nodes/index.js
- /opt/node_modules/@node-red/runtime/lib/index.js
- /opt/node_modules/node-red/lib/red.js
- /opt/node_modules/node-red/red.js

--- a/node-red/package.json
+++ b/node-red/package.json
@@ -45,7 +45,9 @@
     "node-red-node-smooth": "0.1.2",
     "node-red-node-suncalc": "1.0.1",
     "node-red-node-twitter": "1.2.0",
-    "@node-red-contrib-themes/theme-collection": "2.2.0"
+    "@node-red-contrib-themes/theme-collection": "2.2.0",
+    "line-by-line": "0.1.6",
+    "source-map-support": "0.5.21"
   },
   "engines": {
     "node": ">=12"

